packages/react/src/patterns/sections/CTASection/CTASection.js

```diff
@@ -0,0 +1,59 @@
+import { ContentBlock } from '../../sub-patterns/ContentBlock';
+import { ContentItem } from '../../sub-patterns/ContentItem';
```

Comment:
I think you missed exporting the component in `.../packages/react/src/patterns/sections/index.js`

Reply:
Fixed, @annawen1